webapp/src/ts/actions/contacts.ts

```diff
@@ -0,0 +1,26 @@
+import { Store, createAction } from '@ngrx/store';
+import { createSingleValueAction } from './actionUtils';
+
+export const Actions = {
+  updateContactsList: createSingleValueAction('UPDATE_CONTACTS_LIST', 'contacts'),
+  setSelectedReports: createSingleValueAction('SET_SELECTED_REPORTS', 'selected'),
+  resetContactsList: createAction('RESET_CONTACTS_LIST'),
```

Comment:
Can't we just use `updateContactsList([])` instead of this new action? It's already used like this in other parts of the code.

Reply:
Calling `updateContactsList([])` wasn't quite working and resulted in bug. I also like how descriptive calling `resetContactsList` is.